core/src/tile/tileWorker.cpp

```diff
@@ -94,10 +97,10 @@ void TileWorker::run() {
 
             // float loadTime = (float(clock() - begin) / CLOCKS_PER_SEC) * 1000;
             // LOG("loadTime %s - %f", task->tile->getID().toString().c_str(), loadTime);
-
-            m_tileManager.tileProcessed(std::move(task));
```

Comment:
Then TileWorker passes back an empty tile to the TileManager. That could be reloaded once the DataSource changes again. Or am I missing something?